types/highlightjs/tslint.json

```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
[We prefer not to see any rules turned off in new definitions](https://github.com/DefinitelyTyped/DefinitelyTyped#lint). In your description you mentioned that these come with highlightjs but you need them here as well. Is it not feasible to fix these up?